app/views/movies/_movie_partial_loop.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="row tile-container">
   <% movies.each do |movie| %>
     <article class="tile-cover-pic">
-      <!-- MODAL TRIGGER -->
+      <!-- MODAL TRIGGERS -->
```

Comment:
I tried to make these two triggers more visually obvious since I missed it the first time I looked at this page.